src/popup/popup.html

```diff
@@ -82,38 +91,45 @@ <h5 class="b-header margin-bottom-normal">Use</h5>
           ><br />
           <input type="checkbox" id="BY-NC" />
           <label for="BY-NC"
-            ><i class="icon cc-by has-background-white margin-right-smaller"></i
-            ><i class="icon cc-nc has-background-white margin-right-smaller"></i>BY-NC</label
+            ><i class="icon cc-logo has-background-white margin-right-smaller"></i> <i class="icon cc-by has-background-white margin-right-smaller"></i
+            ><i class="icon cc-nc has-background-white margin-right-smaller"></i>BY-NC 
+            <button style="margin-left: 117px ; border: none; background-color: white;"  data-toggle="tooltip" data-placement="right" data-html="true"  title="<b> License CC BY-NC</b><p><i class='icon cc-by'></i> Credit the creator.<br/><i class='icon cc-nc'></i> Noncommercial uses only.</p><p>Read more about the tool <span><a href='#'>here</a></span></p>"><i class="icon question-circle" ></i></button></label
           ><br />
           <input type="checkbox" id="BY-ND" />
           <label for="BY-ND"
-            ><i class="icon cc-by has-background-white margin-right-smaller"></i
-            ><i class="icon cc-nd has-background-white margin-right-smaller"></i>BY-ND</label
+            ><i class="icon cc-logo has-background-white margin-right-smaller"></i> <i class="icon cc-by has-background-white margin-right-smaller"></i
+            ><i class="icon cc-nd has-background-white margin-right-smaller"></i>BY-ND 
+            <button style="margin-left: 116px ; border: none; background-color: white;"  data-toggle="tooltip" data-placement="right" data-html="true"  title="<b> License CC BY-ND</b><p><i class='icon cc-by'></i> Credit the creator.<br/><i class='icon cc-nd'></i> No derivatives or adaptations permitted.</p><p>Read more about the tool <span><a href='#'>here</a></span></p>"><i class="icon question-circle" ></i></button></label
           ><br />
           <input type="checkbox" id="BY-NC-SA" />
           <label for="BY-NC-SA"
-            ><i class="icon cc-by has-background-white margin-right-smaller"></i
+            ><i class="icon cc-logo has-background-white margin-right-smaller"></i> <i class="icon cc-by has-background-white margin-right-smaller"></i
             ><i class="icon cc-nc has-background-white margin-right-smaller"></i
-            ><i class="icon cc-sa has-background-white margin-right-smaller"></i>BY-NC-SA</label
+            ><i class="icon cc-sa has-background-white margin-right-smaller"></i>BY-NC-SA 
+            <button style="margin-left: 73px ; border: none; background-color: white;"  data-toggle="tooltip" data-placement="right" data-html="true"  title="<b> License CC BY-NC-SA</b><p><i class='icon cc-by'></i> Credit the creator.<br/><i class='icon cc-nc'></i> Noncommercial uses only.<br/><i class='icon cc-sa'></i> Share adaptations under the same terms.</p><p>Read more about the tool <span><a href='#'>here</a></span></p>"><i class="icon question-circle" ></i></button></label
```

Comment:
The license information is already present in [helpers.js](https://github.com/creativecommons/ccsearch-browser-extension/blob/master/src/popup/helper.js#L89). Instead of hardcoding the info again in the HTML, we can use JavaScript to add these tooltips dynamically.